zerver/lib/bugdown/__init__.py

```diff
@@ -1200,6 +1201,17 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in realm_slash_commands:
+                escaped = re.escape(("/" + command).lower())
```

Comment:
I'm still not fully convinced that you want to allow any of the following:
- A slash command in the middle of the message
- Multiple slash commands in a single message

For example, could this interpret messages like the following as slash commands?
```
You want to use the command /bin/sh
```

If you really think this is the behavior you want, then can you please find a good place to document the behavior of slash commands?